packages/react-ui/src/theme/themeUtils.js

```diff
@@ -10,3 +10,16 @@ export function getPixelToRem(px) {
 
   return rem;
 }
+
+// Lighten and darken color design formula.
+// The Mui lighten / darken function doesn't get the same color as designers use
+export function getMixedColor(color, baseColor, alpha) {
+  const getRgbaBaseColor = (baseColor, alpha = 1) => {
+    const [r, g, b] = baseColor.match(/\w\w/g).map((x) => parseInt(x, 16));
+    return `rgba(${r},${g},${b},${alpha})`;
+  };
+  const rgbaBaseColorValue = getRgbaBaseColor(baseColor, alpha);
+  const colorResult = `linear-gradient(0deg, ${rgbaBaseColorValue}, ${rgbaBaseColorValue}), ${color}`;
```

Comment:
Ok, I understand you have already validated result with Design. If result is ok for them, I'm ok

Reply:
I've updated vercel again to let them validate it.